examples/oneview_appliance_device_snmp_v1_trap_destinations_facts.yml

```diff
@@ -32,27 +32,26 @@
         params:
           start: 0
           count: 3
-          sort: 'destination:descending'
```

Comment:
any reason for removing this?

Reply:
No specific reason. facts is returning nothing when that filter is used.